proposals/magic-rework.md

```diff
@@ -0,0 +1,115 @@
+# Magic System Rework 2023 #
+
+# **TL;DR** _(Summary)_ #
+Conveying the changes here will take many words. I will try to give a high-level summary of the changes:
+
+- Arcane, Nature, and Spirit Mana Pools are being condensed into a single mana pool.
+- Skill Point costs for Attunement (Mana Pool) are slightly increased across the board
+- Barrier: Spell Aegis, Barkskin, and Toxin Shield merged into a single spell
+- Reflect: Reflect Spell, Stoneskin, Reflect Toxin merged into a single spell
+- Able to have only 1 Barrier and 1 Reflect at a time
+- Introduction of Universal Spells as cross-school spells
+- Introduction of a 10th level of spells
+    - Arcane: Arcane Torrent, Doom 
+    - Nature: Spider Web
+    - Spirit: Death, Life
+- Death and Doom are damage-dealing spells
+- Entrapment is now Entrap Spirit
+
+# The Goal #
+Our goal is to grant casters more compelling customization choices while simultaneously reigning in some of the offensive and defensive power of casters.
+
+This rework may make it seem like things are fundamentally changing in big ways for casters. Many of the changes (such as merging mana pools) is really a formalization of what our players were already doing, ie, dual or tri-school casting anyway.
+
+"Instant Death" take-down spells present a big statting problem for the plot team. Every NPC that goes out has to stack many Resist Magic to challenge the PCs, but doing so ultimately makes high level characters feel ineffective _(I threw 10 Dooms and it resisted all of them!?)_, and low level characters feel useless _(I have too little mana to be useful if everything takes 10 spells to affect)_.
+
+# Single Mana Pool #
+Mana pools are being condensed into a single pool of MP. Instead of having 3 different Attunement skills that each grant mana for a school, there is now a single Attunement skill.
+
+At each rest, the player may allocate their mana as they choose for each school.
+
+> EX: Archibald the Archwizard purchases the Attunement skill 10 times, granting him a total of 100 Mana Points. As he comes into game at the start of the event, he decides to allocate 50 Mana to Arcane, 30 Mana to Spirit, and 20 Mana to Nature. At his next rest, he decides to allocate all 100 Mana Points to Arcane.
+
+Characters may still choose to memorize their spells, and doing so will grant the +10% MP. If a character memorizes in this way, all MP across all schools must be memorized (ie, not allowed to memorize their Arcane spells while Cast on the Fly their Nature spells).
+
+# New Magic Skills #
+**School Initiation** - Replaces the former "mastery" skills. Each purchase allows the character access another school of magic. Characters may only allocate their MP into schools for which they have the associated Initiation skill.
+
+**School Specialization** - New skill. May only be purchased once. Grants the character the ability to cast Level X spells of their chosen school.
```

Comment:
Suggestion:  Use "Level X (Ten)" here since we have a habit of using "X" as a number fill in for things like "mana bolt X".

Reply:
Replaced all instances of "Rank" with "Level", replaced all Roman numerals with Arabic numerals.